### PR TITLE
Update Dutch translations for v3.8.1

### LINKED ISSUE #2022: Update nl_NL app translation for 3.8.1
Hi @henkdegroot, @jerogee,
We are getting ready for the 3.8.1 release. No further changes to translatable strings are currently planned for this release.

We would be happy if you updated the Jamulus software translations for **nl_NL** by **2021-09-30**.

Please
- Update your fork from `jamulussoftware/jamulus` `master` and create a working branch
- Update translations using Qt Linguist in your fork,
- Commit and push your changes and reference this Issue,
- Open a Pull Request before 2021-09-30

Suggested commit and PR message:
```
Update nl_NL app translation for 3.8.1

Fixes #2022
```

This Issue is assigned to multiple people. Please coordinate who will translate what part.

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/TRANSLATING.md).

Thanks for contributing to Jamulus!

### PR DESCRIPTION
**Pull request by henkdegroot**
Update nl_NL app translation for 3.8.1

Fixes #2022

### CODE REVIEW

File: src/res/translation/translation_nl_NL.ts [hunk 1]
```diff
@@ -881,17 +881,17 @@
...
     </message>
     <message>
         <location filename="../../clientdlg.cpp" line="191"/>
         <source>The Ping Time is the time required for the audio stream to travel from the client to the server and back again. This delay is introduced by the network and should be about 20-30 ms. If this delay is higher than about 50 ms, your distance to the server is too large or your internet connection is not sufficient.</source>
-        <translation type="unfinished">De ping-tijd is de tijd die nodig is voor de audiostream om van de client naar de server en terug te reizen. Deze vertraging wordt veroorzaakt door het netwerk en bedraagt ongeveer 20-30 ms. Als deze vertraging hoger is dan circa  50 ms, dan is uw afstand tot de server te groot of is uw internetverbinding niet toereikend.</translation>
+        <translation>De ping-tijd is de tijd die nodig is voor de audiostream om van de client naar de server en terug te reizen. Deze vertraging wordt veroorzaakt door het netwerk en bedraagt ongeveer 20-30 ms. Als deze vertraging hoger is dan circa  50 ms, dan is uw afstand tot de server te groot of is uw internetverbinding niet toereikend.</translation>
```
Comment (jerogee) [inline src/res/translation/translation_nl_NL.ts:889]: Multiple spaces (e.g. "circa   50 ms"). To be sure search / replace across all translation strings?

File: src/res/translation/translation_nl_NL.ts [hunk 2]
```diff
@@ -983,17 +983,17 @@
...
         <location filename="../../clientdlg.cpp" line="159"/>
         <source>Local Jitter Buffer Status LED</source>
-        <translation type="unfinished"></translation>
+        <translation>Lokale jitterbuffer status LED</translation>
     </message>
     <message>
         <location filename="../../clientdlg.cpp" line="160"/>
         <source>The local jitter buffer status LED shows the current audio/streaming status. If the light is red, the audio stream is interrupted. This is caused by one of the following problems:</source>
-        <translation type="unfinished"></translation>
+        <translation>De lokale jitterbuffer status LED geeft de huidige audio/streaming status weer. Indien deze rood is dan is wordt de audio onderbroken. Dit komt door één van de volgende problemen:</translation>
```
Comment (jerogee) [inline src/res/translation/translation_nl_NL.ts:991]: s/is dan is/is dan/

File: src/res/translation/translation_nl_NL.ts [hunk 3]
```diff
@@ -1731,47 +1731,47 @@ We hebben uw kanaal gedempt en &apos;Demp mijzelf&apos; geactiveerd. Los eerst h
...
+        <translation>De buffervertragingsinstelling is een fundamentele instelling van %1. Deze instelling is van invloed op veel verbindingseigenschappen.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="199"/>
         <source>Some sound card drivers do not allow the buffer delay to be changed from within %1. In this case the buffer delay setting is disabled and has to be changed using the sound card driver. On Windows, use the ASIO Device Settings button to open the driver settings panel. On Linux, use the JACK configuration tool to change the buffer size.</source>
-        <translation type="unfinished"></translation>
+        <translation>Sommige stuurprogramma&apos;s van geluidskaarten staan niet toe dat de buffervertraging vanuit %1 kan worden gewijzigd. In dat geval is de buffervertragingsinstelling uitgeschakeld en moet deze worden gewijzigd via het stuurprogramma van de geluidskaart. In Windows drukt u op de knop ASIO Apparaatinstellingen om het instellingenpaneel van het stuurprogramma te openen. In Linux gebruikt u het configuratieprogramma van JACK om de buffergrootte te wijzigen.</translation>
```
Comment (jerogee) [inline src/res/translation/translation_nl_NL.ts:1754]: s/drukt u op/selecteert u/
(more generic, as you can also click a button other than pressing it)

File: src/res/translation/translation_nl_NL.ts [hunk 4]
```diff
@@ -1731,47 +1731,47 @@ We hebben uw kanaal gedempt en &apos;Demp mijzelf&apos; geactiveerd. Los eerst h
...
+        <translation>Als er geen buffergrootte is geselecteerd en de buffervertragingsinstellingen zijn uitgeschakeld, dan betekent dit dat er een niet ondersteunde buffergrootte wordt gebruikt door de driver. %1 zal nog steeds werken met deze instelling maar de werking is wellicht niet optimaal.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="222"/>
         <source>If the buffer delay settings are disabled, it is prohibited by the audio driver to modify this setting from within %1. On Windows, press the ASIO Device Settings button to open the driver settings panel. On Linux, use the JACK configuration tool to change the buffer size.</source>
-        <translation type="unfinished"></translation>
+        <translation>Als de buffervertragingsinstellingen zijn uitgeschakeld, dan staat het stuurprogramma van de geluidskaart het niet toe deze vanuit %1 te wijzigen. In Windows drukt u op de knop ASIO Apparaatinstellingen om het instellingenpaneel van het stuurprogramma te openen. In Linux gebruikt u het configuratieprogramma van JACK om de buffergrootte te wijzigen.</translation>
```
Comment (jerogee) [inline src/res/translation/translation_nl_NL.ts:1764]: s/drukt u op/selecteert u/
(more generic, as you can also click a button other than pressing it)

File: src/res/translation/translation_nl_NL.ts [hunk 5]
```diff
@@ -4043,7 +4043,7 @@ We hebben uw kanaal gedempt en &apos;Demp mijzelf&apos; geactiveerd. Los eerst h
     <message>
         <location filename="../../../windows/sound.cpp" line="554"/>
         <source>Please install an ASIO driver before running %1. If you own a device with ASIO support, install its official ASIO driver. If not, you&apos;ll need to install a universal driver like ASIO4ALL.</source>
-        <translation type="unfinished"></translation>
+        <translation>Installeer een ASIO stuurprogramma voor het starten van %1. Indien u een apparaat heeft met ASIO ondersteuning, installeer dan de officiële ASIO driver voor dit apparaat. Indien u dit niet heeft, dan kunt u een universele driver zoals ASIO4ALL gebruiken.</translation>
```
Comment (jerogee) [inline src/res/translation/translation_nl_NL.ts:4046]: s/universele driver/universeel stuurprogramma/

**Review by jerogee — CHANGES_REQUESTED** [review]
Generally looks fine. Some minor improvements.

File: src/res/translation/translation_nl_NL.ts (hunk 1 above)
Comment (henkdegroot) [reply src/res/translation/translation_nl_NL.ts]: Found 1 other dutch string which contains 2 spaces. So have update that as well. Good spot!

File: src/res/translation/translation_nl_NL.ts (hunk 2 above)
Comment (henkdegroot) [reply src/res/translation/translation_nl_NL.ts]: Done in latest commit.

File: src/res/translation/translation_nl_NL.ts (hunk 3 above)
Comment (henkdegroot) [reply src/res/translation/translation_nl_NL.ts]: Nice one....updated

File: src/res/translation/translation_nl_NL.ts (hunk 4 above)
Comment (henkdegroot) [reply src/res/translation/translation_nl_NL.ts]: Keep it consistent is the goal 👍

File: src/res/translation/translation_nl_NL.ts (hunk 5 above)
Comment (henkdegroot) [reply src/res/translation/translation_nl_NL.ts]: Thanks for the review!

**Review by softins — APPROVED** [review]
Happy to approve, as I'm not a Dutch speaker. Curious though about the change of "View" from "Bekijken" to "Weergave". To me, in this context, "View" is a verb (like "Edit" is), but Google says "Weergave" is a noun. I don't know how other apps translate a "View" menu in Dutch.

**Comment by jerogee** [comment]
@softins , it seems I do not have permissions to approve PRs, which is fine ... and gives you the chance to brush up your Dutch! ;-)

Some other apps, all of which go with the noun. Perhaps the English "View" in menu bars is generally intended to be a noun? Also in other languages we have it as such (e.g. German: "Ansicht", French: "Vue", Italian: "Vista")
- MS, e.g. Office: "View" -> "Beeld" (synonym of "Weergave", so view as a noun)
- Mozilla (Firefox, Thunderbird, etc.): "View" -> "Beeld"
- Google Chrome: "View" -> "Weergave"
- Safari: "View" -> "Weergave"

Either "Weergave" or "Beeld" seems more appropriate (as a noun, or otherwise at least much more common in menu bars) than "Bekijken".

**Comment by softins** [comment]
Unless they have already been done, I think we also need a few updates to `src/res/translation/wininstaller/nl.nsi`, if possible. Some of the other language PRs have them.

**Comment by henkdegroot** [comment]
> Unless they have already been done

Correct, nl jack messages have been done already.

**Pull request merged**